### ISSUE TITLE
SHA256 in homebrew config does not match pkg in Github

### ISSUE THREAD

**Issue by benze** [rodyager/RWTS-PDFwriter#23]
I tried to install RWTS-PDFWriter using Homebrew, but the SHA256 in the brew config does not match the SHA256 of the compiled package.

```
Error: SHA256 mismatch
Expected: 8e653359009f07f107a02b02ac99d198528c87fc280964c772f90fed7d9557d3
  Actual: 3ceb2219db6f279b2c6b1bc53a34b2ac6a919c4650143a825408ba59489e7915
    File: /Users/eric/Library/Caches/Homebrew/downloads/cdbf5b54eb64a97d1ae9c0d0b23cfa5df0641c337171a6ed248204fa4fec2626--RWTS-PDFwriter.pkg
To retry an incomplete download, remove the file above.
```

Is there a way to update the sha in the brew configuration (https://formulae.brew.sh/api/cask/rwts-pdfwriter.json)?

**Comment by rodyager** [comment]
I’m not responsible for the Homebrew configuration. Someone else has repackaged it and not caught up with the fact that I replaced the installer with one that works for M1 machines.

But there is absolutely no reason to install it this way. The easiest way is to download the installer from https://github.com/rodyager/RWTS-PDFwriter/releases/download/v2.0/RWTS-PDFwriter.pkg

If you ever want to unistall it, the package has its own uninstaller that removes every trace. (You find the uninstaller in the Options and Supplies section of the Printer configuration and chosing Utility -> Open Printer Utility.

Rod

On 24 Nov 2022, at 7:13 am, benze ***@***.***> wrote:



I tried to install RWTS-PDFWriter using Homebrew, but the SHA256 in the brew config does not match the SHA256 of the compiled package.

Error: SHA256 mismatch
Expected: 8e653359009f07f107a02b02ac99d198528c87fc280964c772f90fed7d9557d3
  Actual: 3ceb2219db6f279b2c6b1bc53a34b2ac6a919c4650143a825408ba59489e7915
    File: /Users/eric/Library/Caches/Homebrew/downloads/cdbf5b54eb64a97d1ae9c0d0b23cfa5df0641c337171a6ed248204fa4fec2626--RWTS-PDFwriter.pkg
To retry an incomplete download, remove the file above.


Is there a way to update the sha in the brew configuration (https://formulae.brew.sh/api/cask/rwts-pdfwriter.json)?
